compiler/test/runnable_cxx/cpp_abi_tests.d

```diff
@@ -220,6 +222,31 @@ extern(C++)
     }
 }
 
+/*********************************************/
+// https://issues.dlang.org/show_bug.cgi?id=23195
+
+struct FF
+{
+    float x, y;
+
+    ~this() { }
+}
+
+float draw(FF min, FF max)
+{
+    return min.x * 1000 + min.y * 100 + max.x * 10 + max.y;
+}
+
+void test23195()
+{
+    FF a = { 1, 2 };
+    FF b = { 3, 4 };
+    float f = draw(a, b);
+    assert(f == 1234);
```

Comment:
How does this test the ABI?

Reply:
Struct calling convention.

Reply:
I'm not on Windows right now, but I'd guess this test passes before this PR as well. `draw` is not being called from C++, and the destructor does nothing observable, so how does it check whether it's passed by pointer?